packages/components/src/lib/controllers/ui-controller.js

```diff
@@ -278,11 +281,10 @@ if you want to create a different configuration of a UI controller.
       // be added by the components library
       overrideHelp: false,
       /*
-      This is an experimental flag that will force treebank iframe URL to be reset every time a treebank tab is
-      opened. It will force a content of an iframe to be reloaded and thus will solve display issues
-      with a treebank view.
+      How many times to retry and what timout to use within an Arethusa treebank app `refreshView` request.
        */
-      experimentalResetTreebankURL: false,
+      arethusaTbRefreshRetryCount: 50,
+      arethusaTbRefreshDelay: 200,
```

Comment:
They are configurable per page for the embed lib. Because embed lib creates a UI controller, it can provide values of its own. Or are you thinking about something different, like storing it somewhere within data attributes of a page?

Reply:
Ah, I guess you're right. Ok, let's ignore this request. Thanks!